presto-main/src/main/java/com/facebook/presto/sql/gen/PageFunctionCompiler.java

```diff
@@ -157,14 +157,19 @@ public CacheStatsMBean getFilterCache()
     }
 
     public Supplier<PageProjection> compileProjection(SqlFunctionProperties sqlFunctionProperties, RowExpression projection, Optional<String> classNameSuffix)
+    {
+        return compileProjection(sqlFunctionProperties, projection, classNameSuffix, false);
+    }
+
+    public Supplier<PageProjection> compileProjection(SqlFunctionProperties sqlFunctionProperties, RowExpression projection, Optional<String> classNameSuffix, boolean isDisallowLargeByteCodeGeneration)
```

Comment:
Missed this before- this one also has an extra version that doesn't take the boolean argument.

Reply:
This is used by a lot of testing code etc. So it should not need that flag.